bsp/sifive-hifive1-revb/core.dts

```diff
@@ -167,7 +167,7 @@
                         reg = <0x10013000 0x1000>;
                         reg-names = "control";
                         clocks = <&hfclk>;
-                        pinmux = <&gpio0 0x30000 0x30000>;
+                        pinmux = <&gpio0 0x0 0x30000>;
```

Comment:
I assume these has been confirm with the Data Sheet?

Reply:
As per GPIO Muxing for FE310, only PWM IP outputs are mapped to 'IOF1' everything else is mapped to 'IOF0'.
GPIO driver is changed, to allow PWM to be selected from IOF register.

https://github.com/sifive/freedom-metal/blob/sifive-pwm/src/drivers/sifive_gpio0.c#L103